go/cmd/dolt/commands/clone.go

```diff
@@ -189,110 +187,68 @@ func createRemote(ctx context.Context, remoteName, remoteUrl string, params map[
 }
 
 func cloneRemote(ctx context.Context, srcDB *doltdb.DoltDB, remoteName, branch string, dEnv *env.DoltEnv) errhand.VerboseError {
-	var branches []ref.DoltRef
-	if len(branch) > 0 {
-		branches = []ref.DoltRef{ref.NewBranchRef(branch)}
-	} else {
-		var err error
-		branches, err = srcDB.GetBranches(ctx)
+	err := actions.Clone(ctx, srcDB, dEnv.DoltDB)
```

Comment:
Does this mean we lose progress indicators?

Reply:
fixed